### DISCUSSION TITLE
Kairos Image is not getting installed on the Harddisk

### DISCUSSION THREAD

**Discussion by SSWAMIN1SSS** [kairos-io/kairos#1925] [Q&A]
Performed steps as mentioned in the link https://kairos.io/docs/reference/build-from-scratch/
Made Pendrive bootable with the generated iso image from above step.
Installed Kairos using interactive installation mode by selecting installation device as /dev/sda(Harddisk) whereas installation is happening on the pendrive(where iso image is available)
I want to install OS on the Harddisk.
Could anyone help me in resolving this.

**Comment by mauromorales** [comment]
hi @SSWAMIN1SSS, thanks for reaching out. Are you seeing any errors? Or in case you don't could you share the logs

**Reply by SSWAMIN1SSS** [reply]
Will try to capture logs and update in this

**Reply by SSWAMIN1SSS** [reply]
I have a pendrive(which has kairos.iso image) and a SSD Connected, i selected iso in interative installation mode and selected respective device /dev/sda
Sharing the message could you please help me in resolving it
![59a4e328-695a-4f9a-af5e-6a24821381b7](https://github.com/kairos-io/kairos/assets/69335071/a9a9e2e9-2ab6-4d55-ae6d-d0c23da70e65)

**Comment by jimmykarily** [comment]
@SSWAMIN1SSS try to boot in `manual install` mode when you are presented with the grub menu. They run the installation with the `--debug` flag. E.g. `kairos-agent --debug manual-install --device /dev/sda config.yaml | tee output` (make sure you create the config.yaml file first).

You can attach the contents of the `output` file here. To avoid screenshots which are not searchable and hard to read, `scp` the file from the box onto your workstation.
